assignment-1-sql/bdixon/brad_assignment_1.sql

```diff
@@ -0,0 +1,75 @@
+-- Assignment #1
+-- Question 1: Find the most recent completed order where the basket size was > $500.
+
+select id, user_id, completed_at, charge_amount
+from fact_order
+where charge_amount > 500
+and completed_at is not null
+order by completed_at desc
+limit 1
+
+-- Question 2: What GMV have we generated, how many orders were made per week in the 
+-- last 8 weeks in Canada? Order the results from the most recent week to the oldest.
+
+select date_trunc('week',completed_at) as completion_week, sum(f.charge_amount) as total_GMV, count(f.id) as order_count 
+from fact_order f
+    join zones z on f.zone_id = z.id 
+    join regions r on z.region_id = r.id
+where r.country_id = 124
+and completion_week between '2019-09-23' and '2019-11-11'
+group by 1
+order by 1 
```

Comment:
The order here is from oldest to newest rather than newest to oldest as specified in the question.